index.html

```diff
@@ -203,6 +203,23 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </div>
         </section>
+        <section class="contact-secion">
+            <div class="contact">
+                <div class="contact-info">
+                    <h2>Contact me</h2>
+                    <p>if you have an application you are intersted in developing, a feature you that you need built or a project that needs coding. I'd love to help with it!</p>
+                </div>
+                <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] I also tried to submit my information through your form but it didn't send successfully. This was the feedback message below. Kindly fix this so visitors can successfully reach out to you when they fill out the form and submit it :+1: 
- Feedback 
![Screenshot from 2023-03-28 17-57-09](https://user-images.githubusercontent.com/102699958/228328638-e947013c-7cd5-4bb3-8d70-59ebcf0b3211.png)